src/pages/Release.tsx

```diff
@@ -428,6 +433,43 @@ function Release({ releaseInfo }: PageParams) {
                         }}
                       />
                     </motion.div>
+                    {releaseInfo.releaseColumnOptions && releaseInfo.releaseColumnOptions.length > 0 && (
+                      <motion.div variants={variants.fadeInWithLift} className="space-y-4 ">
+                        <Heading level={3}>Release Columns</Heading>
+                        {releaseInfo.releaseColumnOptions.map((releaseOption) => (
+                          <div className="flex flex-row items-center gap-x-2" key={releaseOption.displayName}>
+                            <Input
+                              type="checkbox"
+                              data-testid={`${releaseOption.displayName}-checkbox`}
+                              className={'w-5 rounded'}
+                              checked={selectedReleaseColumns.some(
+                                (column) =>
+                                  column.displayName === releaseOption.displayName &&
+                                  column.queryParamName &&
```

Comment:
Is this expression correct? And what is it for? `column.displayName === releaseOption.displayName && column.queryParamName && releaseOption.queryParamName`?

Reply:
This is to update the checkbox status (checked or unchecked), by comparing two `ReleaseFileOptionFieldsFragment` objects - one in  model data associated with each checkbox with the one in `selectedReleaseColumns`. The comparison compares both `displayName` and `queryParamName` of each `ReleaseFileOptionFieldsFragment` to ensure it is the same object.
Hope that make sense

Reply:
Should it say `column.queryParamName === releaseOption.queryParamName` instead of `column.queryParamName && releaseOption.queryParamName`

Reply:
Ohh,  I didn't see it. You are absolutely right. I will change it now

Reply:
Done 👍